Project/src/main/resources/templates/editor.html

```diff
@@ -28,7 +29,10 @@
         </thead>
         <tbody>
             <tr th:each="article: ${articles}" th:style="${article.isOnTime() ? '' : 'background-color: #FF6961'}">
-                <td th:text="${article.fileName}"></td>
+                <td>
+                    <label th:text="${article.fileName}"></label>
```

Comment:
This is a design choice so do what you will

I think you can just have the filename AS the download href. So the column will just have the filename, if you click it, you download the file. If you feel like what you have currently is better design, just resolve and ask for a re-review

Reply:
good idea